things/advanced-coffee-machine/README.md

```diff
@@ -0,0 +1,24 @@
+# Advanced Coffee Machine Thing
+
+A smart coffee machine simulation for the Eclipse Thingweb Test Things project.
+
+## Description
+
+This advanced coffee machine simulates a device with resource management, drink scheduling, and maintenance features. It is designed for protocol and feature-rich IoT testing.
+
+## Features
+
+-   Resource tracking (water, milk, chocolate, coffee beans)
+-   Drink scheduling and possible drinks list
+-   Maintenance alerts
+-   Make drink and set schedule actions
+-   Out-of-resource event
+-   HTTP protocol implementation (TypeScript)
+
+## Protocol Support
+
+-   HTTP (TypeScript)
```

Comment:
Personally, I would remove any mentioning about TypeScript or JavaScript.

2 lines above is the same in the features list..

Reply:
EDIT: I just saw that @egekorkan mentioned the same already